ocfweb/lab_reservations/mail_templates/user_notification.jinja

```diff
@@ -0,0 +1,12 @@
+Hello {{request.real_name}},
+
+The OCF has received your request to reserve the lab on {{request.date}}
+from {{request.starttime}} to {{request.endtime}} and it is pending staff
+approval. Your request is currently being processed. Should we encounter
+any difficulty or require any additional information, we will send you
+an email. Once we've made a decision, we'll send you an email to let
+you know.
+
+If, for whatever reason, you need to contact us about this reservation
+request, please don't hesitate to send an email to help@ocf.berkeley.edu.
```

Comment:
I think mentioning here that they can just reply to this email (since it's sent from help@ocf.berkeley.edu) would be nice, maybe something like `please don't hesitate to reply to this email or send a new email to help@ocf.berkeley.edu`?

Reply:
++